flake.nix

```
@@ -0,0 +1,31 @@
+{
+  description = "instantNIX";
+
+  nixConfig = {
+    extra-experimental-features = "nix-command flakes";
+
+    extra-substituters = [
+      "https://instantos.cachix.org"
+    ];
+
+    extra-trusted-public-keys = [
+      "instantos.cachix.org-1:6GezJZuxIK59ydzdyoIw6Yr4H6DaCoLzqhPaDhB8v24="
+    ];
+  };
+
+  outputs = {
+    self,
+    nixpkgs,
+  }: let
+    inherit (nixpkgs) lib;
+    systems = lib.platforms.linux;
+    forAllSystems = f: lib.genAttrs systems (system: f system);
+  in {
+    packages = forAllSystems (system:
+      lib.filterAttrs (n: v: lib.isDerivation v)
+      (import ./default.nix {
+        pkgs = nixpkgs.legacyPackages.${system};
```

Sorry, my fault. The flake in Nixpkgs only provides the systems named in [`flake-utils.lib.default-systems`](https://github.com/numtide/flake-utils), i.e. `[ x86_64-linux aarch64-linux x86_64-darwin aarch64-darwin ]`.

For that we might need
```suggestion
        pkgs = nixpkgs.legacyPackages.${system} or (import nixpkgs { inherit system; });
```

---

I see. Thanks for the catch!